crates/bevy_ecs/src/query/filter.rs

```diff
@@ -336,7 +384,32 @@ impl<'a, T: Bundle> Fetch<'a> for WithBundleFetch<T> {
     }
 }
 
+/// A filter that tests if any of the given filters apply.
+///
+/// This is useful for example if a system with multiple components in a query only wants to run
+/// when one or more of the components have changed.
+///
```

Comment:
We should note that filters can be combined by nesting them in a tuple, but that results in an AND behavior.